dev-aws/kafka-shared/customer-billing/invoice-fulfillment/invoice-fulfillment.tf

```diff
@@ -0,0 +1,21 @@
+resource "kafka_topic" "invoice-fulfillment" {
+  name               = "bex.internal.bill_fulfilled"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    # keep data for 7 days
+    "retention.ms" = "604800000"
+    # allow max 1 MB for a message
+    "max.message.bytes" = "1048576"
+    "compression.type" = "zstd"
+    "cleanup.policy"   = "delete"
+  }
+}
+
+module "producer" {
```

Comment:
I think this should have a more meaningful name, like module "invoice_fulfillment_producer"

Reply:
Assumed this was referencing something else and required the same name as the example. I will change it to the more descriptive version.